drevalpy/datasets/curvecurator.py

```diff
@@ -0,0 +1,186 @@
+"""Contains all function required for CurveCurator fitting."""
```

Comment:
Can you add documentation about what curve curator is, link to paper etc ?

Reply:
done